@here/harp-mapview/lib/MapView.ts

```diff
@@ -910,6 +910,7 @@ export class MapView extends THREE.EventDispatcher {
                     : this.m_options.powerPreference
         });
         this.m_renderer.autoClear = false;
+        this.m_renderer.localClippingEnabled = true;
```

Comment:
Please add a comment why this is needed. In which space are the clipping planes at the end. I assume local tile space but would be nice to have it stated here.

Reply:
this will disappear once this change is rebased on top of: https://github.com/heremaps/harp.gl/pull/1030